app/src/main/java/org/mozilla/fenix/search/SearchDialogController.kt

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-package org.mozilla.fenix.searchdialog
+package org.mozilla.fenix.search
```

Comment:
Why is the package named 'search' when we are switching the search fragment to a search dialog?

Reply:
Great question! When we implemented the search dialog, we split it off into it's own folder so it was clear. Since we officially removed all of the old search fragment stuff, we don't need to specify that it's search vs search _dialog_, so I chose to get rid of the search dialog folder and move it into the search folder with the rest of the controllers, etc